deploy/kube-templates/glusterfs-daemonset.yaml

```diff
@@ -23,6 +23,12 @@ spec:
       - image: gluster/gluster-centos:latest
         imagePullPolicy: IfNotPresent
         name: glusterfs
+        env:
+        - name: SYSTEMD_IGNORE_CHROOT
+          value: "1"
+        command:
+        - /usr/lib/systemd/systemd
+        - --system
```

Comment:
Hmm, thanks, I must have read over this. ;-)

It also does not seem to make sense to me (at least it's not justified in the manpage), and I think it must probably be revisited with containers not giving out PID1 necessarily...